controlplane/src/core/util.ts

```diff
@@ -54,6 +56,21 @@ export async function handleError<T extends ResponseMessage>(
   }
 }
 
+export const enrichLogger = async (req: object, ctx: HandlerContext, logger: Logger, authenticator: Authenticator) => {
+  const authContext = await authenticator.extractUserAndOrgId(ctx.requestHeader);
+  return logger.child({
+    service: ctx.service.typeName,
+    method: ctx.method.name,
+    actor: {
+      userId: authContext.userId,
+      organizationId: authContext.organizationId,
+    },
+    req: {
```

Comment:
The request could contain sensitive information and also files. Let's start with the org and user context.

Reply:
The request body should never contain sensitive information. Its public by default.

Reply:
Email is already sensitive. We use that information in several places.